lua/due_nvim.lua

```diff
@@ -75,28 +109,56 @@ function M.setup(c)
   vim.api.nvim_command('autocmd BufEnter ' .. ft .. ' hi def link DueDate ' .. date_hi)
 end
 
+
 function M.draw(buf)
+  local user_time
+  local user_hour
+  local user_min
+  local user_sec
+
+  -- get user default time option
```

Comment:
wouldn't it be better just to give a user an option to implicitly set these nubmers in a string, and then parse that string with the pattern?

Reply:
I agree with this 100% and wanted to go that way. A parsing pattern for a string containing the time would be ideal, imo. As I mentioned above, I have no idea how that works. I tried my hand at a hack based on your examples but had no luck after a minute or two, so I moved on to this way of doing it.